app/assets/stylesheets/shared_styles/elements/buttons.scss

```diff
@@ -0,0 +1,155 @@
+// scss-lint:disable NestingDepth
+
+.sci-btn {
```

Comment:
Wouldn't be better to declare the primary button type as well (as it is was now)?

Reply:
I think add additional classes, will be to complex. Before we never use `.btn`, because we override it. But now we can use our primary class `.sci-btn`.